packages/aws-cdk/lib/init.ts

```diff
@@ -260,6 +262,20 @@ async function postInstallJava(canUseNetwork: boolean) {
   await execute('mvn', 'package');
 }
 
+async function postInstallPython(canUseNetwork: boolean) {
+  if (!canUseNetwork) {
+    print(`Please run ${colors.green('python -m venv .env')}!`);
+    return;
+  }
+
+  print(`Executing ${colors.green('python -m venv .env')}`);
+  try {
+    await execute('python3', '-m venv', '.env');
```

Comment:
Would it make sense to also do the `pip install` stuff here so people can get started quicker?